.github/workflows/changelog.yml

```
@@ -50,10 +51,10 @@ jobs:
   # If you add more jobs, remember to add them to the "needs" array.
   confirmChangelogChecksPassed:
     runs-on: ubuntu-latest
-    name: All tests passed
+    name: All checks passed
```

Good catch on changing this name.

Though it will require an admin to update the `required CI checks` as the system is waiting for the old name.

I would also suggest to name it a bit more specific

```suggestion
    name: All changelog checks passed
```

---

originally I had it exactly like that `All changelog checks passed` but I removed `changelog` word before commit :D